js/script.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
+var prefix = "https://cors-anywhere.herokuapp.com/";
+var tweetLink = "https://twitter.com/intent/tweet?text=";
+var quoteUrl = "https://quotesondesign.com/wp-json/posts?filter[orderby]=rand&filter[posts_per_page]=1";
+
+function getQuote() {
+  fetch(prefix + quoteUrl, { cache: "no-store" })
+      .then(function(resp) {
+          return resp.json();
+      })
+      .then(createTweet);
+}
+
+function createTweet(input) {
+  var data = input[0];
```

Comment:
before you choose first element from response check that response is an array with at least one item